### DISCUSSION TITLE
optix不能安装？

### DISCUSSION THREAD

**Discussion by Extra-Creativity** [xmake-io/xmake#4487] [Q&A] [answered]
输入`xrepo search optix`和`xrepo info optix`有结果，`xrepo install optix`不行？
![2023-12-11T22_56_01](https://github.com/xmake-io/xmake/assets/72960952/eccc312e-3848-4ee0-8a8a-f664adbbdbde)

**Comment by waruqi** [comment, marked as answer]
这个包目前是 fetch only 包，没提供 on_install 安装脚本，只提供了查找脚本，去查找用户安装到系统环境的 optix

如果你想要安装，可以提 pr ，补上 on_install 脚本。。
